src/Erl/Process.purs

```diff
@@ -24,6 +30,14 @@ send p x = Raw.send (runProcess p) x
 
 infixr 6 send as !
 
-spawn :: forall a. (Effect a -> Effect Unit) -> Effect (Process a)
-spawn e = Process <$> Raw.spawn (e Raw.receive)
-
+spawn :: forall a. (SpawnedProcessState a -> Effect Unit) -> Effect (Process a)
+spawn e = Process <$> Raw.spawn (e  { receive: Raw.receive
+                                    , receiveWithTimeout: Raw.receiveWithTimeout
+                                    }
+                                    )
+
+spawnLink :: forall a. (SpawnedProcessState a -> Effect Unit) -> Effect (Process a)
+spawnLink e = Process <$> Raw.spawnLink (e  { receive: Raw.receive
+                                    , receiveWithTimeout: Raw.receiveWithTimeout
+                                    }
+                                    )
```

Comment:
Pedantic comment, can you fiddle the whitespace here, I'd line up the curlies/commas, either fiddly as it is here or with a newline to jump to a standard indent